CHANGELOG.adoc

```diff
@@ -15,6 +15,10 @@
 
 https://github.com/oxidecomputer/dropshot/compare/v0.5.1\...HEAD[Full list of commits]
 
+=== Breaking Changes
+
+* https://github.com/oxidecomputer/dropshot/pull/100[#100] The type used for the "limit" argument for paginated resources has changed.  This limit refers to the number of items that an HTTP client can ask for in a single request to a paginated endpoint.  The limit is now 4294967295, where it may have previously been larger.  This is not expected to affect consumers because this limit is far larger than practical.  For details, see #100.
```

Comment:
I'm not sure if it would cloud the issue, but I might add a parenthetical comment to the actual limit, something like `4294967295 (or u32::MAX - 1)`. The linked issue would have this detail, but it might be helpful to know here something about the _type_ that client code could use to represent this. Just a thought, I'm really not sure it's worth it.

Reply:
Fair point.  In this context, I was really only trying to communicate enough information to let the reader determine _whether_ this might affect them.  I think if it even might affect you, you probably want to read the issue details.  What do you think of that?  (Should I make that clearer?)

Reply:
Yeah, this is exactly what I meant by "clouding the issue," so I'm on board with pointing consumers to more detail in the linked issue :)

Reply:
👍 Thanks for taking a look!